### ISSUE TITLE
Random error - Could not convert to integer

### ISSUE THREAD

**Issue by derekcentrico** [Sonarr/Sonarr#6161]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Reference:  https://github.com/Radarr/Radarr/issues/8166 (closed)

As of today, without any upgrades/changes, an error started appearing:  Unable to communicate with transmission. Could not convert to integer: 2413065027. Path '[130].eta'.

I deleted the container and re-added, same thing.  Nothing with Transmission changed.  Was completely random.

Strangely, the past three weeks two or three episodes kept being readded even after I deleted them from activity and clicked block -and- marked them as unmonitored.  Unsure if that's related to this eventual crap out.

### Expected Behavior

Transmission communication to be working fine. 

### Steps To Reproduce

1.  Sign into the system
2.  See no errors- OS: Ubuntu 22.04.3

### Environment

```markdown
- OS: Ubuntu 22.04.3
- Sonarr: 3.0.10.1567
- Docker Install: Yes  (3.0.10.1567-ls211 by linuxserver.io)
- Using Reverse Proxy:  No
- Browser:  Chrome 118 (not related to UI)
- Database: Unknown
```

### What branch are you running?

Main

### Trace Logs?

[v3.0.10.1567] Newtonsoft.Json.JsonReaderException: Could not convert to integer: 2514860352. Path '[130].eta'. ---> System.OverflowException: Value was either too large or too small for an Int32.
  at System.Convert.ThrowInt32OverflowException () [0x00000] in <d636f104d58046fd9b195699bcb1a744>:0 
  at System.Convert.ToInt32 (System.Int64 value) [0x00012] in <d636f104d58046fd9b195699bcb1a744>:0 
  at System.Int64.System.IConvertible.ToInt32 (System.IFormatProvider provider) [0x00000] in <d636f104d58046fd9b195699bcb1a744>:0 
  at System.Convert.ToInt32 (System.Object value, System.IFormatProvider provider) [0x00003] in <d636f104d58046fd9b195699bcb1a744>:0 
  at Newtonsoft.Json.JsonReader.ReadAsInt32 () [0x00086] in <d47de75a7e3f422ca4ca64a654c80495>:0 
   --- End of inner exception stack trace ---
  at Newtonsoft.Json.JsonReader.ReadAsInt32 () [0x000a8] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.JsonReader.ReadForType (Newtonsoft.Json.Serialization.JsonContract contract, System.Boolean hasConverter) [0x00051] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject (System.Object newObject, Newtonsoft.Json.JsonReader reader, Newtonsoft.Json.Serialization.JsonObjectContract contract, Newtonsoft.Json.Serialization.JsonProperty member, System.String id) [0x00280] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject (Newtonsoft.Json.JsonReader reader, System.Type objectType, Newtonsoft.Json.Serialization.JsonContract contract, Newtonsoft.Json.Serialization.JsonProperty member, Newtonsoft.Json.Serialization.JsonContainerContract containerContract, Newtonsoft.Json.Serialization.JsonProperty containerMember, System.Object existingValue) [0x00161] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal (Newtonsoft.Json.JsonReader reader, System.Type objectType, Newtonsoft.Json.Serialization.JsonContract contract, Newtonsoft.Json.Serialization.JsonProperty member, Newtonsoft.Json.Serialization.JsonContainerContract containerContract, Newtonsoft.Json.Serialization.JsonProperty containerMember, System.Object existingValue) [0x0006d] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList (System.Collections.IList list, Newtonsoft.Json.JsonReader reader, Newtonsoft.Json.Serialization.JsonArrayContract contract, Newtonsoft.Json.Serialization.JsonProperty containerProperty, System.String id) [0x00173] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList (Newtonsoft.Json.JsonReader reader, System.Type objectType, Newtonsoft.Json.Serialization.JsonContract contract, Newtonsoft.Json.Serialization.JsonProperty member, System.Object existingValue, System.String id) [0x000dc] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal (Newtonsoft.Json.JsonReader reader, System.Type objectType, Newtonsoft.Json.Serialization.JsonContract contract, Newtonsoft.Json.Serialization.JsonProperty member, Newtonsoft.Json.Serialization.JsonContainerContract containerContract, Newtonsoft.Json.Serialization.JsonProperty containerMember, System.Object existingValue) [0x0007f] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize (Newtonsoft.Json.JsonReader reader, System.Type objectType, System.Boolean checkAdditionalContent) [0x000db] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.JsonSerializer.DeserializeInternal (Newtonsoft.Json.JsonReader reader, System.Type objectType) [0x00054] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.JsonSerializer.Deserialize (Newtonsoft.Json.JsonReader reader, System.Type objectType) [0x00000] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Linq.JToken.ToObject (System.Type objectType, Newtonsoft.Json.JsonSerializer jsonSerializer) [0x00012] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Linq.JToken.ToObject (System.Type objectType) [0x002ff] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at Newtonsoft.Json.Linq.JToken.ToObject[T] () [0x00000] in <d47de75a7e3f422ca4ca64a654c80495>:0 
  at NzbDrone.Core.Download.Clients.Transmission.TransmissionProxy.GetTorrents (NzbDrone.Core.Download.Clients.Transmission.TransmissionSettings settings) [0x00008] in C:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Download\Clients\Transmission\TransmissionProxy.cs:45 
  at NzbDrone.Core.Download.Clients.Transmission.TransmissionBase.GetItems () [0x00012] in C:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Download\Clients\Transmission\TransmissionBase.cs:36 
  at NzbDrone.Core.Download.TrackedDownloads.DownloadMonitoringService.ProcessClientDownloads (NzbDrone.Core.Download.IDownloadClient downloadClient) [0x0000c] in C:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Download\TrackedDownloads\DownloadMonitoringService.cs:93 

### Anything else?

I didn't change anything.  This started happening randomly.  There are zero torrents being downloaded by Sonarr "activity" and zero torrents related to Sonarr in Transmission at this time.

**Comment by derekcentrico** [comment]
Apologies, also reference Sonarr closed https://github.com/Sonarr/Sonarr/issues/5444 which may be related to this issue anew.

**Comment by stevietv** [comment]
please provide full trace logs, not just the single error line

**Comment by markus101** [comment]
This is already fixed in the v4 beta via https://github.com/Sonarr/Sonarr/pull/5447

**Comment by derekcentrico** [comment]
@markus101 copy, no intent to backport since beta probably still has a ways before stable?

**Comment by markus101** [comment]
No, we're only porting security fixes and fixes for critical issues (like we did for the SQLite fix).
